src/libcommonserver/io/iohelper_win.cpp

```diff
@@ -541,10 +553,12 @@ bool IoHelper::setRights(const SyncPath &path, bool read, bool write, bool exec,
         } else {
             grantedPermission |= FILE_GENERIC_EXECUTE;
         }
-        bool res =
-            setRightsWindowsApi(path, grantedPermission, ACCESS_MODE::SET_ACCESS, ioError, logger()) || _isExpectedError(ioError);
+        bool res = setRightsWindowsApi(path, grantedPermission, ACCESS_MODE::SET_ACCESS, ioError, logger(),
+                                       _setRightsWindowsApiInheritance) ||
+                   _isExpectedError(ioError);
         if (res) {
-            res &= setRightsWindowsApi(path, deniedPermission, ACCESS_MODE::DENY_ACCESS, ioError, logger()) ||
+            res &= setRightsWindowsApi(path, deniedPermission, ACCESS_MODE::DENY_ACCESS, ioError, logger(),
+                                       _setRightsWindowsApiInheritance) ||
```

Comment:
Same

Reply:
As discussed, we may want to change the clang format behaviour in this case.
https://github.com/Infomaniak/desktop-kDrive/pull/115/commits/262fff3cd330c7cc3cc36e8d01b7f60924439b38